src/cdk/coercion/string-array.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/**
+ * Coerces a value to an array of trimmed non-empty strings.
+ * Anything that is not an array or null-ish will be turned into a string via a template literal.
```

Comment:
I would say `via toString()` rather than `via a template literal` because the template literal is just calling `toString` under the hood.

Reply:
I wrote template literal to clarify the behavior for `null` etc which only works this way with literals. But I also explicitly describe the behavior for this case, so I can change it to `toString()` to make it easier to read and understand.

Reply:
Yeah, I think since the function explicitly handles `null` and `undefined` it's the idea most people will understand.